src/Avalonia.Base/Media/TextFormatting/TextLayout.cs

```diff
@@ -35,6 +35,7 @@ public class TextLayout : IDisposable
         /// <param name="letterSpacing">The letter spacing that is applied to rendered glyphs.</param>
         /// <param name="maxLines">The maximum number of text lines.</param>
         /// <param name="textStyleOverrides">The text style overrides.</param>
+        [Obsolete("For binary compatibility only. Should be removed in future. Please use overload with full set of parameters.")]
```

Comment:
Is it possible to make some reminder for cleaning-up these duplicated constructors when it will be possible to make breaking changes (i.e. when v12 will be introduced)? Because they litter the code.

Reply:
`// TODO12: Remove in 12.0.0`